client/src/Components/App/App.js

```diff
@@ -17,24 +17,30 @@ const App = () => {
   const [user, setUser] = useState({id: 1, name: 'KB'});
   const [userLogs, setUserLogs] = useState([]);
   const [feeling, setFeeling] = useState('');
-  const [activity, setActivity] = useState('');
   const [journal, setJournal] = useState('');
-  const [activities, setActivities] = useState(activityData);
   const [filteredActivities, setFilteredActivities] = useState([]);
 
-  const updateActivity = (activity) => {
-    setActivity(activity);
-  }
-  
   const saveNewEntry = (newEntry) => {
     setJournal(newEntry);
   }
 
+  const addEntry = (activity) => {
+    const date = new Date();
+    const newEntry = {
+      user_id: user.id,
+      date: date.toISOString().split('T')[0],
+      feeling: feeling,
+      activity: activity,
+      journal_entry: journal
+    }
+    addNewEntry(newEntry).then(getAllEntries).then(data => setUserLogs(data));
```

Comment:
this is SO CLEAN I'm obsessed. Let's go, perfectly up-to-date data model!!

Reply:
+1 - Super duper fantastic special work here from everyone!!!

Reply:
SO clean!